templates/4a_compensate_image_data.ipynb

```diff
@@ -14,7 +14,7 @@
    "id": "237267d0",
```

Comment:
<p>I think we should have a single folder that contains only finalized rosetta matrices across all of the different cohorts, similar to the panel_files folder. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/212/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>